app/grandchallenge/evaluation/forms.py

```diff
@@ -366,6 +366,15 @@ def __init__(self, *args, user, phase: Phase, **kwargs):  # noqa: C901
                 "uploads.change_userupload",
             ).filter(status=UserUpload.StatusChoices.COMPLETED)
 
+    def clean_phase(self):
+        phase = self.cleaned_data["phase"]
+        if phase.count_valid_archive_items == 0:
+            self.errors["__all__"] = [
```

Comment:
This will wipe out the existing content, I think that `add_error` with `field=None` should be used instead (https://docs.djangoproject.com/en/4.2/ref/forms/api/#django.forms.Form.add_error). If so, could you check for other usages of this pattern in the codebase and correct them too?

Reply:
Found a few, none critical (as in we didn't override important error messages in those places), but updated the usage anyway.

Reply:
Great, and it definitely works as expected with ErrorList? Not too sure how good our test coverage is on that.

Reply:
Yes, I played around with it in the shell. ErrorList is really just a list of error strings and the `add_error` method adds all of them.